homeworks/serhii.ripchanskyi_Ripchanskiy/5-rxjs/app.js

```diff
@@ -0,0 +1,36 @@
+const { fromEvent } = window.rxjs;
```

Comment:
Wrap your code into IIFE to hold your scope clear

https://developer.mozilla.org/en-US/docs/Glossary/IIFE